scripts/html_embeddings/generate_embeddings.py

```diff
@@ -204,10 +232,15 @@ def setup_environment(args: argparse.Namespace) -> Dict[str, Any]:
         logger.info("Try using a Hugging Face model name instead of a local path")
         raise
 
-    return {"logger": logger, "paths": paths}
+    return {"logger": logger, "paths": paths}, product
 
 
-def run_download_step(args: argparse.Namespace, paths: Dict[str, Path], logger) -> bool:
+def run_download_step(
+    args: argparse.Namespace,
+    paths: Dict[str, Path],
```

Comment:
very minor nit: Dict->dict (don't need to import it)

Reply:
:+1: Also did List -> list everywhere